staging/src/k8s.io/apimachinery/pkg/api/apitesting/roundtrip/roundtrip.go

```diff
@@ -103,6 +103,12 @@ var globalNonRoundTrippableTypes = sets.NewString(
 	"DeleteOptions",
 )
 
+func GlobalNonRoundTrippableTypes() interface {
+	Has(item string) bool
```

Comment:
this is pretty clever, but a simple getter that returned sets.String as a copy of globalNonRoundTrippableTypes would be easier to understand